src/graph/UserExecutor.cpp

```diff
@@ -0,0 +1,321 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "graph/UserExecutor.h"
+#include "base/MD5Utils.h"
+
+namespace nebula {
+namespace graph {
+
+CreateUserExecutor::CreateUserExecutor(Sentence *sentence,
+                                       ExecutionContext *ectx) : Executor(ectx) {
+    sentence_ = static_cast<CreateUserSentence*>(sentence);
+}
+
+Status CreateUserExecutor::prepare() {
+    password_ = sentence_->getPassword();
```

Comment:
is the password_ save after encrypted?

Reply:
Yes, what is really stored is the string encrypted by md5Encode.